bucket/msklc.json

```
@@ -0,0 +1,22 @@
+{
+    "version": "1.4",
+    "description": "Create or modify keyboard layouts",
+    "homepage": "https://www.microsoft.com/en-us/download/details.aspx?id=102134",
+    "license": "Freeware",
+    "url": "https://download.microsoft.com/download/6/f/5/6f5ce43a-e892-4fd1-b9a6-1a0cbb64e6e2/MSKLC.exe",
+    "hash": "cd12a6b08066133dc4ad71bdcaccf1492e3d9a1de93cf84b7b2824309f0d7255",
+    "installer": {
+        "script": [
+            "Expand-7zipArchive \"$dir\\MSKLC.exe\" \"$dir\" -ExtractDir 'MSKLC' -Removal",
+            "Expand-MSIArchive \"$dir\\MSKLC.msi\" -Removal",
+            "Remove-Item \"$dir\\setup.exe\""
+        ]
+    },
+    "bin": "bin\\i386\\kbdutool.exe",
+    "shortcuts": [
+        [
+            "MSKLC.exe",
+            "Microsoft Keyboard Layout Creator 1.4"
+        ]
```

```suggestion
        [
            "MSKLC.exe",
            "Microsoft Keyboard Layout Creator"
        ]
```

I think the version number is not needed.

---

The installer does that, but I guess we could "modernize" the name.